src/commcare_cloud/ansible/roles/commcarehq/tasks/celery.yml

```diff
@@ -38,6 +38,4 @@
         PROMETHEUS_PUSHGATEWAY_HOST: "{% if prometheus_celery_pushgateway is defined %}{{ prometheus_celery_pushgateway }}{% endif %}"
         TMPDIR: '{{ encrypted_tmp }}'
         DD_TRACE_ENABLED: "{% if not app_processes_config.datadog_pythonagent %}False{% endif %}"
-        DD_TRACE_DJANGO_ENABLED: "{% if not app_processes_config.datadog_pythonagent %}False{% endif %}"
-        DD_TRACE_DEBUG: "{% if not app_processes_config.datadog_pythonagent %}False{% endif %}"
```

Comment:
@zwets re: https://github.com/dimagi/commcare-cloud/pull/6404

After reading the ddtrace source code and consulting Datadog support (see [commit message](https://github.com/dimagi/commcare-cloud/pull/6700/commits/d0ae7b15d715b8130f1618e6188757af40c119a3)) I concluded that these were unnecessary. Do you agree that these seem unnecessary?